src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -50,6 +50,16 @@
  * <pre>
  * &lt;module name="JavadocTagContinuationIndentation"/&gt;
  * </pre>
+ * <pre>
```

Comment:
`Example:` is missing here and it other sections below

Reply:
done